CHANGES.md

```diff
@@ -6,6 +6,15 @@ Version 0.10.3
 
 To be released.
 
+ -  Fixed a vulnerability of the `IAccountStateDelta.TransferAsset()`'s internal
+    implementation that it had doubled the balance for transferring to themselves.
+    *Since this behavioral change breaks the protocol-level compatibility if any actions
+    in your app had used `IAccountStateDelta.TransferAsset()`, you must make the action
+    to conditionally work as it had done for the hard-coded block index before
+    this patch is applied, or you may be possible to hard-fork the network.*  [[#1134]]
```

Comment:
It seems longer than 80 and it makes check `check-build` failed. 😢

Reply:
74a5111866fcef81b4c94048c34ea67e69ff984b amended.